ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.1.1",
+    "version": "0.2.0",
```

Comment:
Technically this is a breaking change to the 'contract', but since this package is in preview I'm just changing the minor version instead of bumping the major to 1.0